docs/source/release_notes/release-notes.rst

```diff
@@ -67,9 +69,9 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1265>`__)
 
     *    - |devbreak|
-         - 'getAsyncRows' and 'getAsyncRowsMultimap' methods have been removed from generated code.  They do not appear valuable to the API and use a nonintuitive and custom 'AsyncProxy' (also removed).
+         - ``getAsyncRows`` and ``getAsyncRowsMultimap`` methods have been removed from generated code.  They do not appear valuable to the API and use an unintuitive and custom ``AsyncProxy`` that was also removed.
            We believe they are unused by upstream applications, but if you do encounter breaks due to this
-           removal please file a ticket with the dev team for immediate support (as we did not take the time to properly deprecate the methods).
+           removal please file a ticket with the dev team for immediate support since the methods were not properly deprecated.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1689>`__)
 
     *    - |fixed|
```

Comment:
We should highlight that this only affects products using the Timelock server with HTTP/2 configured (and link to docs).